taxcalc/params.json

```diff
@@ -985,9 +985,9 @@
     },
     "_ACTC_ChildNum":{
         "long_name": "Additional child tax credit minimum number of qualified children",
-	    "description": "Only families with this number of qualified children or more, are qualified for additional child tax credit",
-        "irs_ref": "Form , line , ",
-        "notes": "HC",
+	"description": "Families with this number of qualified children or more, are qualified for an alternative way to calculate additional child tax credit",
```

Comment:
On Tue, 4 Aug 2015, Matt Jensen wrote:

> 
> In taxcalc/params.json:
> 
> > @@ -985,9 +985,9 @@
> >      },
> >      "_ACTC_ChildNum":{
> >          "long_name": "Additional child tax credit minimum number of quali
> fied children",
> > -	    "description": "Only families with this number of qualified children 
> or more, are qualified for additional child tax credit",
> > -        "irs_ref": "Form , line , ",
> > -        "notes": "HC",
> > +	"description": "Families with this number of qualified children or more, 
> are qualified for an alternative way to calculate additional child tax credi
> t",

We could be more specific:

"...are qualified for the refundable Additional Child Tax Credit".

dan

> 
> Don't need the comma.
> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVfKx8NJqkGjZLwIEdmlYLcdlWSlKks5okKI-gaJpZM4FhcK0.gif]
> 
> 
>